varcode/infer_effect.py

```diff
@@ -177,7 +177,7 @@ def infer_coding_effect(
     # In case sequence isn't a multiple of 3, then truncate it
     # TODO: if we get a frameshift by the end of a CDS (e.g. in the stop codon)
     # then we should use some of the 3' UTR to finish translation.
-    truncated_variant_cds_seq = variant_cds_seq[:len(variant_cds_seq) / 3 * 3]
+    truncated_variant_cds_seq = variant_cds_seq[:int(len(variant_cds_seq) / 3) * 3]
```

Comment:
These are the cases I'm most worried about in the 2to3 conversion. This one looks correct but be really careful with integer divisions. Once one of those bugs creeps in it'll be very annoying to discover.